### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by smarabattula**
Added test scripts to run succesfully.

### CODE REVIEW

File: test/test_end_point.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+
 from src.end_point import EndPoint
 
 
 class TestEndPoint(unittest.TestCase):
     def setUp(self):
         json_dict = {
             "name": "Get employee details",
-            "_postman_id": "385f7848-62db-4435-b7cf-820c3e7e5097",
+            "_postmian_id": "385f7848-62db-4435-b7cf-820c3e7e5097",
```
Comment (MatthewSills) [inline test/test_end_point.py:14]: is this supposed to be "_postman_id"?

Comment (smarabattula) [reply test/test_end_point.py:14]: Actually yeah, but in latest Python versions the variable isn't showing up in collections. So I changed _postman_id to id and it is running fine.

Comment (smarabattula) [reply test/test_end_point.py:14]: Oh right the name is wrong! Fixed it! Thx

Comment (MatthewSills) [reply test/test_end_point.py:14]: Just asking because it is currently "_postmain_id" not "_postman_id" not sure if that spelling matters though

Comment (smarabattula) [reply test/test_end_point.py:14]: It was a spelling mistake I made. Will fix it in the next commits.

**Pull request merged**